### PR TITLE
Adding documentation to specifically call out support for the Echo Show.

### PR DESCRIPTION
**Pull request by jstreb**

### CODE REVIEW

**Comment by jstreb** [comment]
This is just updates to the readme.md file. Not sure why the build is failing.

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 ### 4.2.1 (next)
 
 * [#284](https://github.com/alexa-js/alexa-app/pull/284): Add support for generating `ask-cli`-compatible schema JSON - [@lazerwalker](https://github.com/lazerwalker).
+* [#292](https://github.com/alexa-js/alexa-app/pull/292): Updating README.md with documentation regarding echo show support. = [@jstreb]
```
Comment (dblock) [inline CHANGELOG.md]: You should remove this. It doesn't match the format, but other than that we don't CHANGELOG README changes.

File: README.md [hunk 2]
```diff
@@ -34,6 +34,7 @@
 * [Custom Directives](#custom-directives)
 * [Dialog](#dialog)
 * [Error Handling](#error-handling)
+* [Show Support](#show-support)
```
Comment (dblock) [inline README.md]: It's an "Echo Show" right? I would say that.

File: README.md [hunk 3]
```diff
@@ -891,6 +892,21 @@ app.error = function(exception, request, response) {
...
+```
+app.post(req, res, type, exception) {  
+  // If the device does not support display directives then remove them from the response
+  if (!system.supportsDisplay(req))) {
+    res.response.response.directives = []
+  }
+}
+```
+
+Please reference [Amazon's documentation](https://developer.amazon.com/docs/custom-skills/display-interface-reference.html#display-template-reference) for the list of supported template markup.
```
Comment (dblock) [inline README.md]: "refer to", not "reference" :)

**Comment by dblock** [comment]
Thanks for doing this! Some minor comments.

**Comment by dblock** [comment]
Looks like our build is broken on node 1.0, maybe @lazerwalker or someone else can help.

File: CHANGELOG.md (hunk 1 above)
Comment (jstreb) [reply CHANGELOG.md]: Perfect. I only added it because of warning in the build.

File: README.md (hunk 2 above)
Comment (jstreb) [reply README.md]: 👍

File: README.md (hunk 3 above)
Comment (jstreb) [reply README.md]: true

**Pull request merged**